e2e-tests/test_integration.py

```diff
@@ -44,6 +53,22 @@ def test_vapi(self):
         url_count = int(match.group(1))
         assert url_count > 1, f"{logfile} indicates only {url_count} URL(s) found, expected more than 1"
 
+        # Verify that ZAP report does not contain alerts for URLs excluded in the scan configuration
+        self.replace_from_yaml(f"{self.tempdir}/rapidast-vapi-configmap-urls-exclusions.yaml")
+        self.replace_from_yaml(f"{self.tempdir}/rapidast-vapi-pod.yaml")
```

Comment:
I'm somewhat surprised you went with extending the current test_vapi test rather than creating a new independent test. Any particular reason for that?

Reply:
My main reason for keeping them in the same test was to first verify that the exclusion alerts appear when no exclusions are applied. That said, I’m happy to move the second part into a separate test, or even split both parts into individual tests if that makes things clearer. I don’t have a strong preference, so if you think it would improve clarity, I'm happy to make that change

Reply:
Ah, I see. I wondered if it was because of resource constraints in the eaas space, or performance (setup/teardown of multiple vapi/rapidast pods). Given that these e2e tests can be heavy (i.e. long runtime), I think it would be good to aim for tests are short in individual runtime, and can be be run concurrently (e.g. via `pytest-xdist`, which we're not using now, but be worthwhile if our number of tests increases). That will also make it more covenient for a dev to run a single test outside of CI.

That said, I don't know if trying to do the above now will hit resource constraint issues, and I know iterating on these test in konflux can be painful. So the current approach is fine for now IMO, thanks for clarifying.